lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -25,18 +25,28 @@ class AppHostInstaller
           #
           attr_reader :name
 
+          # @return [String] the pod the app host installer will be installing within.
+          #
+          attr_reader :pod
+
           # Initialize a new instance
           #
           # @param [Sandbox] sandbox @see #sandbox
           # @param [Pod::Project] project @see #project
           # @param [Platform] platform @see #platform
           # @param [String] name @see #name
+          # @param [String] pod @see #pod
           #
-          def initialize(sandbox, project, platform, name)
+          def initialize(sandbox, project, platform, name, pod)
             @sandbox = sandbox
             @project = project
             @platform = platform
             @name = name
+            @pod = pod
+            target_group = project.pod_group(pod) ||  project.add_pod_group(pod, pod,
```

Comment:
discussed IRL, this should already exist, so probably just need to add a `before` block to tests